.travis.yml

```diff
@@ -34,7 +34,9 @@ before_script:
 - ./etc/testing/travis_install_batch_deps.sh
 script:
 - make lint vet
-- travis_wait 30 ./etc/testing/travis.sh
+# Don't use travis_wait, which can cause logs to take a long time to flush to s3
+# - travis_wait 30 ./etc/testing/travis.sh
+- ./etc/testing/travis.sh
```

Comment:
Might as well just delete the line. It's easy enough to recover if we want it back.